src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -192,6 +193,17 @@ public WorkspaceModel CurrentSpace
             get { return model.CurrentWorkspace; }
         }
 
+        /// <summary>
+        /// Controls if the the ML data ingestion pipeline is beta from feature flag
+        /// </summary>
+        public bool MLDataIngestionPipelineTest
```

Comment:
Does this need to be public? Consider renaming to `IsMLDataIngestionPipelineinBeta`?

Reply:
Renamed it and made it internal. Not making it public as we are not showing the toggle in preferences window.